src/main/java/org/olf/folio/order/OrderImport.java

```diff
@@ -96,6 +104,7 @@ private void importOnePurchaseOrderPerMarcRecord(MarcReader reader, Results resu
 			try {
 				Record record = reader.next();
 				MarcToFolio mappedMarc = Config.getMarcMapping(record);
+				mappedMarc.addSessionContext(session);
```

Comment:
Of course I could just pass the particular session property I need (the username) instead of the whole session.  This approach presumes there could be other session data that may be useful in the future.  Maybe not likely.